apps/gpc-prover-client/package.json

```diff
@@ -1,18 +1,23 @@
 {
-  "name": "web",
+  "name": "gpc-prover-client",
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3002",
     "build": "next build",
     "start": "next start",
     "lint": "eslint . --max-warnings 0"
   },
   "dependencies": {
-    "@repo/ui": "*",
+    "@pcd/gpc": "^0.1.0",
```

Comment:
We're going to want to update these on occasion.  I think this'll become a useful test case when we release new NPM Packages, to make sure the example app still builds and deploys.

Reply:
cool, let me know if there's anything I need to do in this repo to enable that

Reply:
I'm thinking it's probably just a manual process.  There might be some way to automate it with a workflow, but I'm not familiar enough with that to do it easily.